docs/examples/error_handler/error_handler_0/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "error-handler-0"
+dynamic = ["version"]
+description = "This is just an error handler example package"
+readme = "README.rst"
+license = "Apache-2.0"
+requires-python = ">=3.7"
+authors = [
+  { name = "OpenTelemetry Authors", email = "cncf-opentelemetry-contributors@lists.cncf.io" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+  "opentelemetry-sdk ~= 1.3",
+]
+
+[project.entry-points.opentelemetry_error_handler]
+error_handler_0 = "error_handler_0:ErrorHandler0"
+
+[tool.hatch.version]
+path = "src/error_handler_0/error_handler_0/version.py"
```

Comment:
Should  this be `src/error_handler_0/version.py`?

Reply:
Ah yes

Reply:
Fixed!